From 1d0f24e077f80d123e53e476fcd57293b82052d5 Mon Sep 17 00:00:00 2001
From: Jonathan Kang <jonathankang@gnome.org>
Date: Fri, 10 May 2019 15:10:55 +0800
Subject: [PATCH] network: Fix mtu label not requesting enough space

Remove the size group where mtu label belongs, so that it can request
enough space when it becomes visible. Since the size group is removed,
filler widget is no longer useful. Remove it as well.
---
 .../network/connection-editor/ethernet-page.ui | 18 ------------------
 1 file changed, 18 deletions(-)

diff --git a/panels/network/connection-editor/ethernet-page.ui b/panels/network/connection-editor/ethernet-page.ui
index eb1bfb06a..650f72e8a 100644
--- a/panels/network/connection-editor/ethernet-page.ui
+++ b/panels/network/connection-editor/ethernet-page.ui
@@ -199,18 +199,6 @@
         <property name="height">1</property>
       </packing>
     </child>
-    <child>
-      <object class="GtkLabel" id="filler">
-        <property name="visible">True</property>
-        <property name="can_focus">False</property>
-      </object>
-      <packing>
-        <property name="left_attach">2</property>
-        <property name="top_attach">0</property>
-        <property name="width">1</property>
-        <property name="height">1</property>
-      </packing>
-    </child>
     <child>
       <placeholder/>
     </child>
@@ -227,10 +215,4 @@
       <placeholder/>
     </child>
   </object>
-  <object class="GtkSizeGroup" id="sizegroup1">
-    <widgets>
-      <widget name="label_mtu"/>
-      <widget name="filler"/>
-    </widgets>
-  </object>
 </interface>
-- 
2.21.0

